README.md

```diff
@@ -12,6 +12,7 @@ The following actions are supported:
 - Seek controls (<< and >>)
 - Mouse cursor movement
 - Left-mouse-button click using tapping
+- Rigth-mouse-button double click using double tapping
```

Comment:
nit: Spelling of `right`. Should it read instead `Right-mouse-button click using double tapping`? I believe the `double` is extra.

Reply:
Ok I'll surely fix that , thank-you very much for your guidance.